v1/src/main/java/com/google/cloud/teleport/templates/SpannerVectorEmbeddingExport.java

```diff
@@ -0,0 +1,314 @@
+    },
+    // ToDo: Update the documentation
+    documentation =
+        "https://cloud.google.com/dataflow/docs/guides/templates/provided/cloud-spanner-vectors-to-cloud-storage",
+    contactInformation = "https://cloud.google.com/support",
+    requirements = {
+      "The Cloud Spanner database must exist.",
+      "The output Cloud Storage bucket must exist.",
+      "In addition to the Identity and Access Management (IAM) roles necessary to run Dataflow jobs, you must also have the <a href=\"https://cloud.google.com/spanner/docs/export#iam\">appropriate IAM roles</a> for reading your Cloud Spanner data and writing to your Cloud Storage bucket."
+    })
+public class SpannerVectorEmbeddingExport {
+
+  private static final Logger LOG = LoggerFactory.getLogger(SpannerVectorEmbeddingExport.class);
+
+  /** Custom PipelineOptions. */
+  public interface SpannerToVectorEmbeddingJsonOptions extends PipelineOptions {
+    @TemplateParameter.ProjectId(
+        order = 10,
+        description = "Cloud Spanner Project Id",
+        helpText = "The project id of the Cloud Spanner instance.")
+    ValueProvider<String> getSpannerProjectId();
+
+    void setSpannerProjectId(ValueProvider<String> value);
+
+    @TemplateParameter.Text(
+        order = 20,
+        regexes = {".+"},
+        description = "Read data from Cloud Spanner Instance",
+        helpText = "Instance of requested table.")
+    ValueProvider<String> getSpannerInstanceId();
+
+    @SuppressWarnings("unused")
+    void setSpannerInstanceId(ValueProvider<String> spannerInstanceId);
+
+    @TemplateParameter.Text(
+        order = 30,
+        regexes = {".+"},
+        description = "Read data from Cloud Spanner Database ",
+        helpText = "Database of requested table.")
+    ValueProvider<String> getSpannerDatabaseId();
+
+    @SuppressWarnings("unused")
+    void setSpannerDatabaseId(ValueProvider<String> spannerDatabaseId);
+
+    @TemplateParameter.Text(
+        order = 40,
+        regexes = {"^.+$"},
+        description = "Spanner Table",
+        helpText = "Spanner Table to read from")
+    ValueProvider<String> getSpannerTable();
+
+    void setSpannerTable(ValueProvider<String> table);
+
+    @TemplateParameter.Text(
+        order = 50,
+        description = "Columns to Export from Spanner Table",
+        helpText =
+            "Comma separated list of columns which are required for Vertex AI Vector Search Index."
+                + " If the column names don't precisely align with the Vertex AI Vector Search Index input structure, you can establish column mappings using aliases."
+                + " For e.g. if you have columns id and my_embedding i.e. the id column matches what vertex expects but the embedding column is named differently,"
+                + " you can specify the following `id, my_embedding:embedding`.")
```

Comment:
The example is not clearly explaining which side of the `:` is column name and which side of the `:` is alias. Explain aliasing in an example and mention about `id` and `embedding` in another sentence.